db/schema.rb

```diff
@@ -10,18 +10,35 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171109075512) do
+ActiveRecord::Schema.define(version: 20171109125137) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "ingredients", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.boolean "status", default: true
+    t.integer "price"
+    t.string "name"
+  end
+
+  create_table "product_lines", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "product_id"
+    t.integer "ingredient_id"
+  end
+
   create_table "products", force: :cascade do |t|
     t.string "name"
     t.integer "price"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.text "description"
     t.string "image"
+    t.string "of_type", default: "dish"
+    t.boolean "status", default: true
```

Comment:
I'm not sure what this attribute is going to be used for. Please explain

Reply:
Status is used for availability. Probs should have called it available though

Reply:
I think that's a good idea.